x-pack/platform/plugins/shared/inference/server/tasks/nl_to_esql/doc_base/suggestions.ts

```diff
@@ -13,6 +13,11 @@ const suggestions: Suggestion[] = [
       return ['BUCKET'];
     }
   },
+  (keywords) => {
+    if (keywords.includes('TO_DATETIME')) {
```

Comment:
I always suspected that AI was just a bunch of `if` statements 😄

Reply:
Sometimes we have to cheat a bit!